vms/metervm/state_syncable_vm.go

```diff
@@ -78,3 +79,37 @@ func (vm *blockVM) GetStateSummary(ctx context.Context, height uint64) (block.St
 	vm.blockMetrics.getStateSummary.Observe(duration)
 	return summary, nil
 }
+
+func (vm *blockVM) BackfillBlocksEnabled(ctx context.Context) (ids.ID, uint64, error) {
+	if vm.ssVM == nil {
+		return ids.Empty, 0, block.ErrStateSyncableVMNotImplemented
+	}
+
+	start := vm.clock.Time()
+	blkID, blkHeight, err := vm.ssVM.BackfillBlocksEnabled(ctx)
+	end := vm.clock.Time()
+	duration := float64(end.Sub(start))
+	if err != nil {
+		vm.blockMetrics.backfillBlocksEnabled.Observe(duration)
+		return ids.Empty, blkHeight, err
+	}
+	vm.blockMetrics.backfillBlocksEnabled.Observe(duration)
+	return blkID, blkHeight, nil
+}
+
+func (vm *blockVM) BackfillBlocks(ctx context.Context, blocks [][]byte) (ids.ID, uint64, error) {
+	if vm.ssVM == nil {
+		return ids.Empty, 0, block.ErrStateSyncableVMNotImplemented
+	}
+
+	start := vm.clock.Time()
+	nextWantedBlkID, nextWantedBlkHeight, err := vm.ssVM.BackfillBlocks(ctx, blocks)
+	end := vm.clock.Time()
+	duration := float64(end.Sub(start))
+	if err != nil {
+		vm.blockMetrics.backfillBlocks.Observe(duration)
+		return nextWantedBlkID, nextWantedBlkHeight, err
+	}
+	vm.blockMetrics.backfillBlocks.Observe(duration)
+	return nextWantedBlkID, nextWantedBlkHeight, nil
+}
```

Comment:
this is all we need to extende the interface in the metervm, so I figured I coded it in this first PR to reduce diffs down the line